index.ts

```diff
@@ -289,46 +294,45 @@ function variablizeColor(p_color, id) {
   return `var(${varStr}, ${color})`;
 }
 
-function variablizeStrokeWidth(strokeWidth, id) {
+function variablizeStrokeWidth(strokeWidth: string, id: number): string {
   const varStr = id === 1 ? `--${opts.strokeWidths.name}` : `--${opts.strokeWidths.name}-${id}`;
   if(opts.strokeWidths.customVars && opts.strokeWidths.customVars[strokeWidth]) {
     return `var(--${opts.strokeWidths.customVars[strokeWidth]}, var(${varStr}, ${strokeWidth}))`
   }
   return `var(${varStr}, ${strokeWidth})`;
 }
 
-function variablizeTransitionStyle(style) {
+function variablizeTransitionStyle(style: TransitionOptions) {
```

Comment:
this is the style string from the current dom element, has nothing to do with transition options